# Salmon_HK_SingleCellSeq
Code for running analysis and figures from article: 
Mapping the cellular landscape of Atlantic salmon head kidney by single cell and single nucleus transcriptomics. Doi: https://doi.org/10.1016/j.fsi.2024.109357

Abstract:
Single-cell transcriptomics is the current gold standard for global gene expression profiling, not only in mammals and model species, but also in non-model fish species. This is a rapidly expanding field, creating a deeper understanding of tissue heterogeneity and the distinct functions of individual cells, making it possible to explore the complexities of immunology and gene expression on a highly resolved level. In this study, we compared two single cell transcriptomic approaches to investigate cellular heterogeneity within the head kidney of healthy farmed Atlantic salmon (Salmo salar). We compared 14,149 cell transcriptomes assayed by single cell RNA-seq (scRNA-seq) with 18,067 nuclei transcriptomes captured by single nucleus RNA-Seq (snRNA-seq). Both approaches detected eight major cell populations in common: granulocytes, heamatopoietic stem cells, erythrocytes, mononuclear phagocytes, thrombocytes, B cells, NK-like cells, and T cells. Four additional cell types, endothelial, epithelial, interrenal, and mesenchymal cells, were detected in the snRNA-seq dataset, but appeared to be lost during preparation of the single cell suspension submitted for scRNA-seq library generation. We identified additional heterogeneity and subpopulations within the B cells, T cells, and endothelial cells, and revealed developmental trajectories of heamatopoietic stem cells into differentiated granulocyte and mononuclear phagocyte populations. Gene expression profiles of B cell subtypes revealed distinct IgM and IgT-skewed resting B cell lineages and provided insights into the regulation of B cell lymphopoiesis.. The analysis revealed eleven T cell sub-populations, displaying a level of T cell heterogeneity in salmon head kidney comparable to that observed in mammals, including distinct subsets of cd4/cd8-negative T cells, such as tcrÎ³ positive, progenitor-like, and cytotoxic cells. Although snRNA-seq and scRNA-seq were both useful to resolve cell type-specific expression in the Atlantic salmon head kidney, the snRNA-seq pipeline was overall more robust in identifying several cell types and subpopulations. While scRNA-seq displayed higher levels of ribosomal and mitochondrial genes, snRNA-seq captured more transcription factor genes. However, only scRNA-seq-generated data was useful for cell trajectory inference within the myeloid lineage. In conclusion, this study systematically outlines the relative merits of scRNA-seq and snRNA-seq in Atlantic salmon, enhances understanding of teleost immune cell lineages, and provides a comprehensive list of markers for identifying major cell populations in the head kidney with significant immune relevance.
